README.md

```diff
@@ -6,6 +6,28 @@ go-ipfix is an IPFIX library that can be used to implement an IPFIX exporter, wh
 ## Try it out
 This IPFIX library can be used to build an exporter. Please check out the [exporter tests](https://github.com/vmware/go-ipfix/blob/main/pkg/exporter/process_test.go) to get an idea on how to build exporter on top of TCP and UDP transport protocols given a IPFIX collector.
 
+### Deploy stand alone IPFIX collector
+To deploy a standalone go-ipfix collector, which is used to decode and log the IPFIX records, you can use:
+```
+cd <directory containing this README file>/build/yamls
+kubectl apply -f ./ipfix-collector.yaml
+```
+The collector's port and protocol can be configured by using the command:
+
+```
+cd <directory containing this README file>/hack
+./generate-manifest-collector.sh --mode dev --addr "port:proto" > ../build/yamls/ipfix-collector.yaml
```

Comment:
I think using separate params here would be better: `--port 4739 --protocol tcp`

Reply:
Yes. I was doing this because originally we have three arguments `addr:port:proto`. I feel like four separate arguments (including --mode) might be too long. Then we remove the `addr` arg. Now I'm waiting for Srikar's opinion on whether we should keep the port argument or not.